test/unit/L2/l2LPTGateway.test.ts

```diff
@@ -349,7 +349,7 @@ describe('L2 Gateway', function() {
 
       it('should fail to tranfer', async () => {
         await expect(
-            l2Gateway.outboundTransfer(
+            l2Gateway['outboundTransfer(address,address,uint256,bytes)'](
```

Comment:
Not the prettiest, but I guess this is how Typechain handles overloaded Solidity functions. I suppose it makes sense because TS doesn't have native function overloading either.

Reply:
Thankfully autocomplete works with this haha!